frontend/js/utils/api.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+const baseUrl = window.location.host;
+
+const apiUrls = {
+  battleDetail: (id) => `api/battle/${id}`,
+  currentUser: 'api/user/',
+};
+
+const createTeamUrl = (id) => {
+  const url = `http://${baseUrl}/team/${id}/create/`;
```

Comment:
We are using `django-js-reverse` to pass urls from django to react. Instructions on how to set this here: https://github.com/ierror/django-js-reverse#use-the-urls-in-javascript

Reply:
thanks :D